python/ray/tests/test_actor_failures.py

```diff
@@ -207,6 +207,46 @@ def check_reconstructed():
     assert wait_for_condition(check_reconstructed)
 
 
+def test_actor_caller_reconstruction(ray_start_regular):
```

Comment:
Can you also add a test case for a task caller?

Reply:
added, thanks